Aquarius/Aquarius.cpp

```diff
@@ -1,5 +1,8 @@
+#define AQ_ASSERT_ENABLE
```

Comment:
Eventually these kind of definitions will be set based on the build mode (debug/release). Thats some house keeping we'll have to do later on

Reply:
Yeah that is something that can occur within CMake